.rubocop_todo.yml

```diff
@@ -1,6 +1,6 @@
 # This configuration was generated by
-# `rubocop --auto-gen-config --auto-gen-only-exclude --no-exclude-limit --no-offense-counts --no-auto-gen-timestamp`
```

Comment:
As someone trying to destroy this file, I find the full file lists to be more useful. It's easier to see where the most work has to be done, and/or identify things to just disable because we might not care or have a real pref.

Reply:
If you run it locally, you'll see that it has changed several of the `Max` values, like AbcComplexity to full lists now> there is probably an alternate with a specific limit value that might keep the current approach, without folding down this one

Reply:
Ah, I see what you mean. Looks like this issue changed (arguably fixed?) how it works - https://github.com/rubocop/rubocop/pull/12902 - and now, yes, lots of changes on regen - https://github.com/mastodon/mastodon/compare/main...mjankowski:mastodon:rubocop-regen-with-fixed-no-exclude


Hmmm ... I think I'd still prefer to leave `--no-exclude-limit` in place for this file, but perhaps in advance of this 1.64.0 regen we should set some actual values in the main config (maybe with notes about the default value next to our override value) for things where we previously relying on this behavior to put the `Max` values into the `_todo`?

Reply:
Yeah, I'll probably send in a cleanup PR for this one that is changing. I think you already submitted one to turn it off anyway, but I'm not sure there is a specific style or not

Reply:
> Yeah, I'll probably send in a cleanup PR for this one that is changing.

Sounds good!

> I think you already submitted one to turn it off anyway

Yup, https://github.com/mastodon/mastodon/pull/30043 - whenever I go to "fix" these, it always looks worse in my opinion. I generally try to avoid guard clauses, but I think at the lint level it's better to just not have a policy and use discretion case by case.

I also have a WIP branch which does a little more in the way of setting specific values for some of the rspec cops, and could add similar for some of the Metrics ones. That branch though is contingent on getting https://github.com/mastodon/mastodon/pull/30407 or similar merged first.

Reply:
The linked issues here have either been merged/closed at this point, and we're also a patch version bump up in rubocop (and maybe solved a few more things?)

I think at this point doing what you've done here (remove the `--no-exclude-limit`) actually does make sense. Want to rebase/regen this with higher version and see if anything else changes?

Reply:
Thanks for push -- I endorse the current version of this.